.gitignore

```diff
@@ -6,6 +6,7 @@ node_modules
 test-results
 __diff_output__
 .jest-cache
+__tests__/src/__image_snapshots__/integration-snap.png
```

Comment:
I dont think this is needed

Reply:
Maybe not, but it's something generated by the test run so it seems unnecessary to keep it in git